public/app/features/variables/adhoc/picker/AdHocPicker.tsx

```diff
@@ -10,20 +10,22 @@ import { addFilter, changeFilter, removeFilter } from '../actions';
 import { REMOVE_FILTER_KEY } from './AdHocFilterKey';
 import { AdHocFilterRenderer } from './AdHocFilterRenderer';
 
-interface OwnProps extends VariablePickerProps<AdHocVariableModel> {}
+const mapDispatchToProps = {
+  addFilter,
+  removeFilter,
+  changeFilter,
+};
 
-interface ConnectedProps {}
+const mapStateToProps = (state: StoreState) => ({});
 
-interface DispatchProps {
-  addFilter: typeof addFilter;
-  removeFilter: typeof removeFilter;
-  changeFilter: typeof changeFilter;
-}
+const connector = connect(mapStateToProps, mapDispatchToProps);
```

Comment:
```suggestion
const connector = connect(null, mapDispatchToProps);
```
I think this should work without having to declare empty `mapStateToProps`.

Reply:
Will have a look, thanks for noticing!

Reply:
Works great, thank you!